app/src/main/AndroidManifest.xml

```diff
@@ -17,10 +17,6 @@
             android:noHistory="true"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme.NoActionBar" >
-            <intent-filter>
```

Comment:
Lets have this back as it was. The `SpashScreenActivity` does some data preloading on the background.